labgrid/remote/client.py

```diff
@@ -373,7 +373,8 @@ def get_acquired_place(self, place=None):
 
     async def del_place(self):
         """Delete a place from the coordinator"""
-        name = self.args.place
+        place = self.get_idle_place()
```

Comment:
We shouldn't use pattern matching when deleting to avoid accidents.

Reply:
This has been fixed.